Makefile

```diff
@@ -357,6 +357,9 @@ alpine: ## Creat an "alpine" OCI bundle in target
 
 # CI Commands
 
+.PHONY: ci-test
+ci-test: pki config test
```

Comment:
i'm not a huge fan (personal opinion) of having make targets just for convenience.  if the GHA should run `make pki config test` then let's do that so it's explicit what it's doing.  it keeps the local development workflow simple as `make test` should always pass and i don't need to think about what `make ci-test` is or why it's different.